docsbox/docs/views.py

```diff
@@ -122,6 +122,46 @@ def post(self, file_id):
             return abort(500, e, request, traceback.format_exc())
         return response
 
+class DocumentConvertViewV2(Resource):
+
+    def post(self, file_id):
+        """
+        Creates converting task of given file
+        """
+        response = {}
+        try:
+            if request.files and "file" in request.files:
+                filename = request.files["file"].filename
+                mimetype = get_file_mimetype_from_data(request.files["file"], filename)
+                file_path = store_file(request.files["file"], filename)
+                if mimetype not in app.config["CONVERTABLE_MIMETYPES"]:
```

Comment:
In order to keep the implementation uniform, let's only respond with "queued" status from this API and move all corrupted handling to the worker, even though if we had the file available here. Just to keep the integration easier.